tests/network/OWNERS

```diff
@@ -6,3 +6,4 @@ reviewers:
   - Anatw
   - sbahar619
```

Comment:
Well, while you are here - will you please also remove `sbahar619` from the list, as he is (sadly) no longer part of the team.
If you do that, don't forget to change the commit/PR message accordingly (something like "update network reviewers members").
Thank you